js/app.js

```diff
@@ -37,56 +54,49 @@ Question.prototype = {
     }
 };
 
-// Details for each question
-var typeface = new Array(); //GARAMOND
-typeface[0] = new Font(
-    ".garamond",
-    ".garamond",
-    ["Verdana", "Garamond", "Monotype"],
-    "Garamond"
-);
-console.log(typeface);
-
-typeface[1] = new Font(  //HELVETICA
-    ".helvetica",
-    ".helvetica",
-    ["Baskerville", "Sans-serif", "Helvetica"],
-    "Helvetica"
-);
-console.log(typeface);
-
-typeface[2] = new Font( //TIMES
-    ".times",
-    ".times",
-    ["Goudy Old Style", "Serifa", "Times"],
-    "Times"
-);
-console.log(typeface);
-
-typeface[3] = new Font( //VERDANA
-    ".verdana",
-    ".verdana",
-    ["Verdana", "Verdy", "Script"],
-    "Verdana"
-);
-console.log(typeface);
-
-typeface[4] = new Font( //ARIAL
-    ".arial",
-    ".arial",
-    ["Comic Sans", "Trade Gothic", "Arial"],
-    "Arial"
-);
-console.log(typeface);
```

Comment:
That looks pretty good.  I also noticed that I messed up my code.  
```javascript
answer: function(choice) { 
    this.answer = choice;
}
```
should probably be
```javascript
setAnswer: function(choice) { 
    this.answer = choice;
}
```
I'm guessing you noticed this issue when trying to run question.answer(choice) more than one time?  This will throw an error on the second time because the `answer` function gets overridden when you run `this.answer = choice`.  Sorry!